C++/Logger/conversion/ConvertibleTo.h

```diff
@@ -6,8 +6,11 @@ namespace Toolbox::Log::Conversion
 
 #include "../metaprogramming/value_type_is.h"
 
-template <typename ResultType>
-class ConvertibleTo : public value_type_is <ResultType>
```

Comment:
`value_type` represents the parameterized type, in this case their are n >= 1 parameterized types.  We could make it the first one.